mne/time_frequency/tfr.py

```diff
@@ -619,6 +625,7 @@ def _tfr_aux(method, inst, freqs, decim, return_itc, picks, average,
     out = _compute_tfr(data, freqs, info['sfreq'], method=method,
                        output=output, decim=decim, **tfr_params)
     times = inst.times[decim].copy()
+    info['sfreq'] /= decim.step
```

Comment:
It would be better to do this inside `_compute_tfr` where the objects are created (you can just do `info['sfreq'] = sfreq` in there) so that the division is done in one place. With your change the decim/sfreq logic now lives both in this function and `_compute_tfr`

Reply:
(for @agramfort since he did this :) )

Reply:
@agramfort in that case I suspect you'll also need a `if decim.step is not None:` (or to change `_check_decim` to set `step` to `1` even if it's `None`) as we needed this conditional above